pynbody/halo/ahf.py

```diff
@@ -66,8 +66,13 @@ def __init__(self, sim, make_grp=None, get_all_parts=None, use_iord=None, ahf_ba
         self._dosort = dosort
         self._only_stat = only_stat
 
+        RamsesTF = isinstance(sim.ancestor, (snapshot.ramses.RamsesSnap))
```

Comment:
Why is there anything special to do with RAMSES outputs? Is it because we need to convert the RAMSES output to tipsy/gadget output which is then fed into AHF?
If so, it may be cleaner to have a more generic solution for this, rather than having RAMSES-related code in the AHF frontends. I'd be happy to discuss this!

Reply:
Yeah thats exactly why this came about, but this is also because of the way you point to the ramses files when you load them in via pynbody you point to the level in which it expects the AHF files. For example with gadget/tipsy this isnt a problem because you point directly to the simulation files where as with ramses its a folder you point to, and so things could get very cluttered very fast

Reply:
I see. What do you think of the two suggestions I am making?